### ISSUE TITLE
Unable to locate a class or view for component [heroicon-o-check-circle].

### ISSUE THREAD

**Issue by PouyaPour** [laravel-filament/filament#2677]
### Package

filament/filament

### Package Version

v2.12.9

### Laravel Version

9.16.0

### PHP Version

8.1.6

### Bug description

I created new project with Laravel v9 and Filament v2. After login I face with this error
`InvalidArgumentException
PHP 8.1.6
9.16.0
Unable to locate a class or view for component [heroicon-o-check-circle].`
Please Help me

### Relevant log output

```shell
[2022-06-06 15:53:42] local.ERROR: Unable to locate a class or view for component [heroicon-o-check-circle]. {"view":{"view":"/var/www/vendor/filament/filament/resources/views/components/notification.blade.php","data":[]},"userId":1,"exception":"[object] (Spatie\\LaravelIgnition\\Exceptions\\ViewException(code: 0): Unable to locate a class or view for component [heroicon-o-check-circle]. at /var/www/vendor/laravel/framework/src/Illuminate/View/Compilers/ComponentTagCompiler.php:299)
[stacktrace]
#0 /var/www/vendor/laravel/framework/src/Illuminate/View/Compilers/ComponentTagCompiler.php(211): Illuminate\\View\\Compilers\\ComponentTagCompiler->componentClass()
#1 /var/www/vendor/laravel/framework/src/Illuminate/View/Compilers/ComponentTagCompiler.php(196): Illuminate\\View\\Compilers\\ComponentTagCompiler->componentString()
#2 [internal function]: Illuminate\\View\\Compilers\\ComponentTagCompiler->Illuminate\\View\\Compilers\\{closure}()
#3 /var/www/vendor/laravel/framework/src/Illuminate/View/Compilers/ComponentTagCompiler.php(191): preg_replace_callback()
#4 /var/www/vendor/laravel/framework/src/Illuminate/View/Compilers/ComponentTagCompiler.php(89): Illuminate\\View\\Compilers\\ComponentTagCompiler->compileSelfClosingTags()
#5 /var/www/vendor/laravel/framework/src/Illuminate/View/Compilers/ComponentTagCompiler.php(76): Illuminate\\View\\Compilers\\ComponentTagCompiler->compileTags()
#6 /var/www/vendor/laravel/framework/src/Illuminate/View/Compilers/BladeCompiler.php(414): Illuminate\\View\\Compilers\\ComponentTagCompiler->compile()
#7 /var/www/vendor/laravel/framework/src/Illuminate/View/Compilers/BladeCompiler.php(247): Illuminate\\View\\Compilers\\BladeCompiler->compileComponentTags()
#8 /var/www/vendor/laravel/framework/src/Illuminate/View/Compilers/BladeCompiler.php(166): Illuminate\\View\\Compilers\\BladeCompiler->compileString()
#9 /var/www/vendor/livewire/livewire/src/ComponentConcerns/RendersLivewireComponents.php(49): Illuminate\\View\\Compilers\\BladeCompiler->compile()
#10 /var/www/vendor/laravel/framework/src/Illuminate/View/View.php(139): Livewire\\LivewireViewCompilerEngine->get()
#11 /var/www/vendor/laravel/framework/src/Illuminate/View/View.php(122): Illuminate\\View\\View->getContents()
#12 /var/www/vendor/laravel/framework/src/Illuminate/View/View.php(91): Illuminate\\View\\View->renderContents()
#13 /var/www/vendor/laravel/framework/src/Illuminate/View/Concerns/ManagesComponents.php(104): Illuminate\\View\\View->render()
#14 /var/www/vendor/filament/filament/src/../resources/views/components/notification-manager.blade.php(27): Illuminate\\View\\Factory->renderComponent()
#15 /var/www/vendor/livewire/livewire/src/ComponentConcerns/RendersLivewireComponents.php(83): include('...')
#16 /var/www/vendor/livewire/livewire/src/ComponentConcerns/RendersLivewireComponents.php(84): Filament\\Pages\\Dashboard->Livewire\\ComponentConcerns\\{closure}()
#17 /var/www/vendor/livewire/livewire/src/ComponentConcerns/RendersLivewireComponents.php(59): Livewire\\LivewireViewCompilerEngine->evaluatePath()
#18 /var/www/vendor/laravel/framework/src/Illuminate/View/View.php(139): Livewire\\LivewireViewCompilerEngine->get()
#19 /var/www/vendor/laravel/framework/src/Illuminate/View/View.php(122): Illuminate\\View\\View->getContents()
#20 /var/www/vendor/laravel/framework/src/Illuminate/View/View.php(91): Illuminate\\View\\View->renderContents()
#21 /var/www/vendor/laravel/framework/src/Illuminate/View/Concerns/ManagesComponents.php(104): Illuminate\\View\\View->render()
#22 /var/www/vendor/filament/filament/src/../resources/views/components/page.blade.php(77): Illuminate\\View\\Factory->renderComponent()
#23 /var/www/vendor/livewire/livewire/src/ComponentConcerns/RendersLivewireComponents.php(83): include('...')
#24 /var/www/vendor/livewire/livewire/src/ComponentConcerns/RendersLivewireComponents.php(84): Filament\\Pages\\Dashboard->Livewire\\ComponentConcerns\\{closure}()
#25 /var/www/vendor/livewire/livewire/src/ComponentConcerns/RendersLivewireComponents.php(59): Livewire\\LivewireViewCompilerEngine->evaluatePath()
#26 /var/www/vendor/laravel/framework/src/Illuminate/View/View.php(139): Livewire\\LivewireViewCompilerEngine->get()
#27 /var/www/vendor/laravel/framework/src/Illuminate/View/View.php(122): Illuminate\\View\\View->getContents()
#28 /var/www/vendor/laravel/framework/src/Illuminate/View/View.php(91): Illuminate\\View\\View->renderContents()
#29 /var/www/vendor/laravel/framework/src/Illuminate/View/Concerns/ManagesComponents.php(104): Illuminate\\View\\View->render()
#30 /var/www/vendor/filament/filament/src/../resources/views/pages/dashboard.blade.php(4): Illuminate\\View\\Factory->renderComponent()
#31 /var/www/vendor/livewire/livewire/src/ComponentConcerns/RendersLivewireComponents.php(83): include('...')
#32 /var/www/vendor/livewire/livewire/src/ComponentConcerns/RendersLivewireComponents.php(84): Filament\\Pages\\Dashboard->Livewire\\ComponentConcerns\\{closure}()
#33 /var/www/vendor/livewire/livewire/src/ComponentConcerns/RendersLivewireComponents.php(59): Livewire\\LivewireViewCompilerEngine->evaluatePath()
#34 /var/www/vendor/laravel/framework/src/Illuminate/View/View.php(139): Livewire\\LivewireViewCompilerEngine->get()
#35 /var/www/vendor/laravel/framework/src/Illuminate/View/View.php(122): Illuminate\\View\\View->getContents()
#36 /var/www/vendor/laravel/framework/src/Illuminate/View/View.php(91): Illuminate\\View\\View->renderContents()
#37 /var/www/vendor/livewire/livewire/src/Component.php(235): Illuminate\\View\\View->render()
#38 /var/www/vendor/livewire/livewire/src/HydrationMiddleware/RenderView.php(14): Livewire\\Component->output()
#39 /var/www/vendor/livewire/livewire/src/LifecycleManager.php(141): Livewire\\HydrationMiddleware\\RenderView::dehydrate()
#40 /var/www/vendor/livewire/livewire/src/Macros/livewire-view-component.blade.php(3): Livewire\\LifecycleManager->initialDehydrate()
#41 /var/www/vendor/laravel/framework/src/Illuminate/Filesystem/Filesystem.php(107): require('...')
#42 /var/www/vendor/laravel/framework/src/Illuminate/Filesystem/Filesystem.php(108): Illuminate\\Filesystem\\Filesystem::Illuminate\\Filesystem\\{closure}()
#43 /var/www/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(58): Illuminate\\Filesystem\\Filesystem->getRequire()
#44 /var/www/vendor/livewire/livewire/src/ComponentConcerns/RendersLivewireComponents.php(69): Illuminate\\View\\Engines\\PhpEngine->evaluatePath()
#45 /var/www/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(61): Livewire\\LivewireViewCompilerEngine->evaluatePath()
#46 /var/www/vendor/livewire/livewire/src/ComponentConcerns/RendersLivewireComponents.php(35): Illuminate\\View\\Engines\\CompilerEngine->get()
#47 /var/www/vendor/laravel/framework/src/Illuminate/View/View.php(139): Livewire\\LivewireViewCompilerEngine->get()
#48 /var/www/vendor/laravel/framework/src/Illuminate/View/View.php(122): Illuminate\\View\\View->getContents()
#49 /var/www/vendor/laravel/framework/src/Illuminate/View/View.php(91): Illuminate\\View\\View->renderContents()
#50 /var/www/vendor/laravel/framework/src/Illuminate/Http/Response.php(69): Illuminate\\View\\View->render()
#51 /var/www/vendor/laravel/framework/src/Illuminate/Http/Response.php(35): Illuminate\\Http\\Response->setContent()
#52 /var/www/vendor/laravel/framework/src/Illuminate/Routing/Router.php(833): Illuminate\\Http\\Response->__construct()
#53 /var/www/vendor/laravel/framework/src/Illuminate/Routing/Router.php(802): Illuminate\\Routing\\Router::toResponse()
#54 /var/www/vendor/laravel/framework/src/Illuminate/Routing/Router.php(725): Illuminate\\Routing\\Router->prepareResponse()
#55 /var/www/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(141): Illuminate\\Routing\\Router->Illuminate\\Routing\\{closure}()
#56 /var/www/vendor/filament/filament/src/Http/Middleware/MirrorConfigToSubpackages.php(26): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#57 /var/www/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Filament\\Http\\Middleware\\MirrorConfigToSubpackages->handle()
#58 /var/www/vendor/filament/filament/src/Http/Middleware/DispatchServingFilamentEvent.php(15): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#59 /var/www/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Filament\\Http\\Middleware\\DispatchServingFilamentEvent->handle()
#60 /var/www/vendor/laravel/framework/src/Illuminate/Routing/Middleware/SubstituteBindings.php(50): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#61 /var/www/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\Routing\\Middleware\\SubstituteBindings->handle()
#62 /var/www/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/VerifyCsrfToken.php(78): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#63 /var/www/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\Foundation\\Http\\Middleware\\VerifyCsrfToken->handle()
#64 /var/www/vendor/laravel/framework/src/Illuminate/Session/Middleware/AuthenticateSession.php(59): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#65 /var/www/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\Session\\Middleware\\AuthenticateSession->handle()
#66 /var/www/vendor/laravel/framework/src/Illuminate/Auth/Middleware/Authenticate.php(44): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#67 /var/www/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\Auth\\Middleware\\Authenticate->handle()
#68 /var/www/vendor/laravel/framework/src/Illuminate/View/Middleware/ShareErrorsFromSession.php(49): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#69 /var/www/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\View\\Middleware\\ShareErrorsFromSession->handle()
#70 /var/www/vendor/laravel/framework/src/Illuminate/Session/Middleware/StartSession.php(121): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#71 /var/www/vendor/laravel/framework/src/Illuminate/Session/Middleware/StartSession.php(64): Illuminate\\Session\\Middleware\\StartSession->handleStatefulRequest()
#72 /var/www/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\Session\\Middleware\\StartSession->handle()
#73 /var/www/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/AddQueuedCookiesToResponse.php(37): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#74 /var/www/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\Cookie\\Middleware\\AddQueuedCookiesToResponse->handle()
#75 /var/www/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/EncryptCookies.php(67): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#76 /var/www/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\Cookie\\Middleware\\EncryptCookies->handle()
#77 /var/www/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(116): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#78 /var/www/vendor/laravel/framework/src/Illuminate/Routing/Router.php(726): Illuminate\\Pipeline\\Pipeline->then()
#79 /var/www/vendor/laravel/framework/src/Illuminate/Routing/Router.php(703): Illuminate\\Routing\\Router->runRouteWithinStack()
#80 /var/www/vendor/laravel/framework/src/Illuminate/Routing/Router.php(667): Illuminate\\Routing\\Router->runRoute()
#81 /var/www/vendor/laravel/framework/src/Illuminate/Routing/Router.php(656): Illuminate\\Routing\\Router->dispatchToRoute()
#82 /var/www/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(167): Illuminate\\Routing\\Router->dispatch()
#83 /var/www/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(141): Illuminate\\Foundation\\Http\\Kernel->Illuminate\\Foundation\\Http\\{closure}()
#84 /var/www/vendor/livewire/livewire/src/DisableBrowserCache.php(19): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#85 /var/www/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Livewire\\DisableBrowserCache->handle()
#86 /var/www/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TransformsRequest.php(21): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#87 /var/www/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/ConvertEmptyStringsToNull.php(31): Illuminate\\Foundation\\Http\\Middleware\\TransformsRequest->handle()
#88 /var/www/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\Foundation\\Http\\Middleware\\ConvertEmptyStringsToNull->handle()
#89 /var/www/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TransformsRequest.php(21): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#90 /var/www/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TrimStrings.php(40): Illuminate\\Foundation\\Http\\Middleware\\TransformsRequest->handle()
#91 /var/www/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\Foundation\\Http\\Middleware\\TrimStrings->handle()
#92 /var/www/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/ValidatePostSize.php(27): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#93 /var/www/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\Foundation\\Http\\Middleware\\ValidatePostSize->handle()
#94 /var/www/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/PreventRequestsDuringMaintenance.php(86): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#95 /var/www/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\Foundation\\Http\\Middleware\\PreventRequestsDuringMaintenance->handle()
#96 /var/www/vendor/laravel/framework/src/Illuminate/Http/Middleware/HandleCors.php(49): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#97 /var/www/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\Http\\Middleware\\HandleCors->handle()
#98 /var/www/vendor/laravel/framework/src/Illuminate/Http/Middleware/TrustProxies.php(39): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#99 /var/www/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\\Http\\Middleware\\TrustProxies->handle()
#100 /var/www/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(116): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#101 /var/www/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(142): Illuminate\\Pipeline\\Pipeline->then()
#102 /var/www/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(111): Illuminate\\Foundation\\Http\\Kernel->sendRequestThroughRouter()
#103 /var/www/public/index.php(52): Illuminate\\Foundation\\Http\\Kernel->handle()
#104 {main}
```

**Comment by PouyaPour** [comment]
When I comment line 41 
```
<x-heroicon-o-check-circle class="shrink-0 w-6 h-6" x-show="notification.status === 'success'" x-bind="iconClasses"/>
of code/vendor/filament/filament/resources/views/components/notification.blade.php
```
and line 25
```
 <x-heroicon-o-chevron-down :class="\Illuminate\Support\Arr::toCssClasses([
                    'w-3 h-3 text-gray-600 transition-all',
                    'dark:text-gray-300' => config('filament.dark_mode'),
                ])" x-bind:class="$store.sidebar.groupIsCollapsed(label) || '-rotate-180'" x-cloak />
```
of code/vendor/filament/filament/resources/views/components/layouts/app/sidebar/group.blade.php
The error is fixed

**Comment by ruerdev** [comment]
I just got the same error when deploying to production. Turns out that I run `php artisan icons:cache` when deploying. As is stated [here](https://github.com/blade-ui-kit/blade-icons#caching), you'll not be able to use any icons that are added to the package after you've run `php artisan icons:cache`.

Clearing your icon cache: `php artisan icons:clear` will solve your problem. Make sure to cache your icons again later.
